source/simulate/profile-simulation.md

```diff
@@ -0,0 +1,647 @@
+113        16     785874.0   49117.1     98.5          [individu(ressource, three_previous_months, options = [ADD]) for ressource in ressources_a_inclure]
+148        31     777545.0   25082.1     99.0          chomage_imposable = individu("chomage_imposable", period)
+134        37     857364.0   23172.0     99.5          csg_deductible_chomage = individu("csg_deductible_chomage", period)
+52         37     820912.0   22186.8     97.9          taux_csg_remplacement = individu("taux_csg_remplacement", period)
+28         25    1534376.0   61375.0     99.4          rfr = individu.foyer_fiscal("rfr", period=period.n_2)
+2208        4    1418229.0  354557.2     89.8          rni = foyer_fiscal('rni', period)
+1082        3    2746777.0  915592.3     99.9          rng = foyer_fiscal('rng', period)
+1067        3    2763749.0  921249.7     99.8          rbg = foyer_fiscal('rbg', period)
+1028        3    2757742.0  919247.3     99.9          revenu_categoriel = foyer_fiscal('revenu_categoriel', period)
+983         3    2659102.0  886367.3     99.0          rev_cat_tspr = foyer_fiscal('revenu_categoriel_tspr', period)
+617         3    2701246.0  900415.3    100.0          tspr_i = foyer_fiscal.members('traitements_salaires_pensions_rentes', period)
+556         3    2674322.0  891440.7     99.3          revenu_assimile_salaire_apres_abattements = individu('revenu_assimile_salaire_apres_abattements', period)
+416         4    2702072.0  675518.0    100.0          revenu_assimile_salaire = individu('revenu_assimile_salaire', period)
+401         4    2579118.0  644779.5     97.2          salaire_imposable = individu('salaire_imposable', period, options = [ADD])
+```
+
+From `salaire_imposable`, we can follow two different branches:
+
+1. The `indemnite_residence` branch:
+
+```
+Line #   Hits    Time        Per Hit    % Time         Line Contents
+====================================================================
+
+199        48    1951692.0   40660.2     59.3          indemnite_residence = individu('indemnite_residence', period)
+737        48    1615270.0   33651.5     73.0          _P = parameters(period)
+```
+
+Let's find out where `parameters` is defined by modifying the code as follows:
+
+```diff
+class indemnite_residence(Variable):
+    value_type = float
+    entity = Individu
+    label = "Indemnité de résidence des fonctionnaires"
+    definition_period = MONTH
+    set_input = set_input_divide_by_period
+
+    def formula(individu, period, parameters):
++       breakpoint()
+```
+
+Then running `openfisca test` with [The Python Debugger](https://docs.python.org/3/library/pdb.html):
+
+```
+$ openfisca test --pdb tests/formulas/irpp_prets_participatifs.yaml
+
+...
+
+(Pdb) parameters.__qualname__
+'TaxBenefitSystem.get_parameters_at_instant'
+```
+
+2. The `cotisations_salariales` branch:
```

Comment:
For some reason the `2` here shows as `1` in html. Not only here, but I noticed in [`From Law to code/Parameters/How to write a new parameter`](https://openfisca.org/doc/coding-the-legislation/legislation_parameters.html)as well in the published doc 🤔

Reply:
Nice catch, my last commit fixes the ordered lists :)